src/.nuget/packages.targets

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <UsingTask TaskName="GetBuildUid" TaskFactory="CodeTaskFactory" AssemblyName="Microsoft.Build.Tasks.v12.0">
+        <ParameterGroup>
+            <BuildUid ParameterType="System.String" Output="true" />
+        </ParameterGroup>
+        <Task>
+            <Code>
+                BuildUid = System.Guid.NewGuid().ToString().Replace("-", "").Substring(0, 8);
+            </Code>
+        </Task>
+    </UsingTask>
+    <Target Name="BuildNuGetPackages" AfterTargets="Build">
+        <!-- This is to fool the machine local DNX cache so each build is treated as unique. %USERPROFILE%\.dnx -->
+        <GetBuildUid>
+            <Output PropertyName="BuildUid" TaskParameter="BuildUid" />
+        </GetBuildUid>
+
+        <!-- Property needed for creating nupkgs -->
+        <PropertyGroup>
+            <ToolchainMilestone Condition="'$(ToolchainMilestone)'==''">prerelease</ToolchainMilestone>
+            <ToolchainVersion>1.0.3-$(ToolchainMilestone)-00002</ToolchainVersion>
+            <RuntimeSdkVersion>$(ToolchainVersion)</RuntimeSdkVersion>
+            <NuPkgRid>$(NuPkgRuntimeOS)-$(NuPkgRuntimePlatform)</NuPkgRid>
+            
+            <ToolchainPackageName>Microsoft.DotNet.ILCompiler</ToolchainPackageName>
+            
+            <RuntimeSdkPackageNameSuffix Condition="'$(Configuration)'=='Debug'">.Debug</RuntimeSdkPackageNameSuffix>
+            <RuntimeSdkPackageName>Microsoft.DotNet.ILCompiler.SDK$(RuntimeSdkPackageNameSuffix)</RuntimeSdkPackageName>
+
+            <!-- Uniqify package name per machine-local DNX cache for development purposes -->
+            <ToolchainPackageName Condition="'$(ToolchainMilestone)'=='testing'">$(ToolchainPackageName).$(BuildUid)</ToolchainPackageName>
+            <RuntimeSdkPackageName Condition="'$(ToolchainMilestone)'=='testing'">$(RuntimeSdkPackageName).$(BuildUid)</RuntimeSdkPackageName>
+
+            <LibraryPrefix Condition="'$(OSEnvironment)'=='Unix'">$(LibraryPrefix)lib</LibraryPrefix>
```

Comment:
I would suggest that LibraryPrefix condition is inverted to be set for non-Windows platforms. This will prevent additional changes required if a new non-Windows/non-Unix OS is supported in future.